extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php

```diff
@@ -140,6 +144,58 @@ private function renderInfoboxHero( $data ) {
 	}
 
 	/**
+	 * renders infobox gallery component
+	 *
+	 * @param array $data - array of images
+	 *
+	 * @return string
+	 */
+	private function renderGallery( $data ) {
+		$helper = new PortableInfoboxRenderServiceHelper();
+
+		$galleryData = [];
+
+		$cover = array_shift($data);
+		$galleryData['cover'] = $helper->extendImageData( $cover );
+		$galleryData['extras'] = array_map([$helper,'extendImageData'], $data);
+
+		$markup = $this->renderItem( 'gallery', $galleryData );
+
+		return $markup;
+	}
+
+	/**
+	 * temporary function to provide mock data for a gallery
+	 *
+	 * @return array
+	 */
+	private function mockGalleryData() {
```

Comment:
This is a mock so doesn't matter (does not have to be fixed) but we prefer to prefix names of this kind of methods with word "get" getMockedGaleryData, etc.